types/stripe/index.d.ts

```diff
@@ -2100,8 +2100,50 @@ declare namespace Stripe {
                  * URL to redirect to upon success
                  */
                 success_url: string;
+                /**
```

Comment:
Needs a newline above.

Reply:
Fixed that, linting this large file is killing me, for some reason prettier is not behaving right